08-rails/active-record-models.md

```diff
@@ -5,137 +5,133 @@
 ```
 
 ## AR Class Methods: Finding and Organizing Records
 ActiveRecord class methods are generally for locating & reading rows from the database:
 
 ```ruby
-# return a collection of all rows in the students table
-students = Student.all
+# return a collection of all rows in the books table
+books = Book.all
 
-# return the first student (lowest id)
-student = Student.first
+# return the first book (lowest id)
+book = Book.first
 
-# return the first student (lowest id) named Libby
-libby = Student.find_by(name: 'Libby')
+# return an individual record using the id
+Book.find(1) # returns the Book with ID of value 1
 
-# return students in a specific order
-alphabetical_students = Student.order(:name)
+# return the first book (lowest id) authored by Roxane Gay
+roxane_books = Book.find_by(author: 'Roxane Gay')
 
 # the AR DSL is structured to use chaining to create more specific queries
-# here we find all students that prefer apply pie
+# here we find all books by Roxane Gay
 # *and* return them in reverse alphabetical order
-team_apple = Student.where(pie: "Apple").order(name: :desc)
+roxane_books_ordered = Book.where(author: "Roxane Gay").order(title: :desc)
 
 # provide compound conditions to do more specific queries
-rosa_and_apple = Student.where(name: "Rosa", pie: "apple")
+specific_book = Book.where(author: "Roxane Gay", title: "Bad Feminist")
 
 # use `not` to exclude values from results
-not_apple = Student.where.not(pie: "apple")
-rosa_but_not_apple = Student.where(name: "Rosa").where.not(pie: "apple")
+not_roxane = Book.where.not(author: "Roxane Gay")
+feminists_not_roxane = Book.where(title: "Feminism").where.not(author: "Roxane Gay")
 
-# we can use `find` to pull a bunch of records at once too
+# we can use `where` to pull a bunch of records at once
 ids = [1,2,3]
-Student.find(ids) # returns an array of Student records
+Book.where(id: ids)
 
-# ...but I think using `where` to do this is a more semantic
-Student.where(id: ids)
+# return books in a specific order
+alphabetical_books = Book.order(:author)
 ```
 
 ### Parameter Binding with Active Record
 
 Active Record uses something called _parameter binding_ to take our requests and convert them into SQL, the nearly universal language of databases and (most importantly) make queries more secure (google research: [_sql injection_](https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=sql%20injection)), and to improve the reusability of queries. This idea is very important in Rails, and AR provides it.
 
-If you write code using the `where` syntax above in the rails console (like `Student.where(name: "Rosa")`), Active Record _does parameter binding automatically_. Active Record can also provides manual parameter binding:
+If you write code using the `where` syntax above in the rails console (like `Book.where(name: "Rosa")`), Active Record _does parameter binding automatically_. Active Record can also provides manual parameter binding:
```

Comment:
We should say `Book.where(author: "Roxane Gay")` to keep with the text explanation that follows.